docs/adr/0001-update-major-version-with-uswds-minor-version.md

```diff
@@ -0,0 +1,100 @@
+
+Therefore, any time we decide to update the version of USWDS used in ReactUSWDS, we should expect to perform the following changes:
+
+- Update the USWDS version listed in both `devDependencies` and `peerDependencies`
+- Perform any API or implementation changes to existing components detailed in the USWDS release notes
+  - For example, from [2.10.1](https://github.com/uswds/uswds/releases/tag/v2.10.1):
+    > Use proper semantic markup for footer title: Now we're using a p instead of an h3 for usa-footer\_\_logo-heading since the name of the project or agency is hot a heading in this context.
+
+The questions this ADR seeks to answer are:
+
+- When performing a USWDS version update as described above, what kind of release should that result in for ReactUSWDS? (i.e., major / minor / patch)
+  - At minimum, this update will always require users to also update their version of USWDS.
+- Should USWDS version updates be timed and released in sequence with the components they implement?
+  - For example, should we aim to release all components introduced in USWDS 2.9.0 before releasing the update to 2.10.0 and any of its components?
+
+For reference, USWDS does mention semantic versioning in [their own release process documentation](https://github.com/uswds/uswds/wiki/Release-process#versioning), and documents breaking changes (constituting major releases) as follows:
+
+> In most software projects, the "public API" corresponds to a single set of programming constructs, such as public classes or functions. Because the USWDS consists of tightly-bound HTML, CSS, and JavaScript, we must consider any "breaking" change to any of these as a change to the public API. For example, any of the following should trigger a major version increment:
+>
+> - Changing the name of any .usa- class name (documented or not)
+> - Changing the way in which elements with .usa- class names are structured in HTML
+> - Changing the HTML "API" for any of our interactive components, such as the accordion
+
+## Decision Drivers
+
+- We want to provide as many USWDS components as possible to the widest audience as possible, taking into account users who may be version-locked to a legacy USWDS version for whatever reason
+- We don't want to impede development and release of USWDS components, and we aim to "catch up" to the current USWDS release eventually
+
+## Considered Options
+
+- Bump the ReactUSWDS major version when updating USWDS minor & major versions
+- Bump the ReactUSWDS version with the same release type as USWDS
```

Comment:
I'm comfortable with option 1 or option 2.

We know that `uswds` _does_ release presentational changes that are sometimes not mentioned in the release notes, which we _could_ consider breaking changes.

Reading the `uswds` release process makes me actually lean towards option 2. If our library is simply a react-wrapper for `uswds`, is there really any value to us being _more strict_ than `usdws`? Option 1 would mean that projects depending on both libraries would have minor version updates for `uswds` and a major version updates for `react-uswds` for the same changes. Specifying the package.json dependencies on a consumer project would be weird in that context. For example, if you use `^2.10.0` for `uswds`, you wouldn't have a good equivalent for `react-uswds` without also opening yourself up to other breaking changes. It kind of sounds like we're wanting consumers to manually manage the versions of USWDS and React, so maybe that concern isn't as big of a deal as I'm making it sound like.

Related note (to option 2): 

I would've thought based on 

> the following should trigger a major version increment:
> - Changing the way in which elements with .usa- class names are structured in HTML

that the 2.10.1 change (a _patch_ version change!)

> Now we're using a `p` instead of an `h3` for usa-footer\_\_logo-heading

would have constituted a **major** version change.

But ultimately for me its coming back to "What does being _more_ strict than the library we're wrapping ultimately buy us?". If consumers are expected to manually manage these versions, the stability of this library's versions isn't a big of a concern.

Reply:
Great point I like how you framed that.